src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/BackgroundDocumentGeneratorTest.cs

```diff
@@ -37,6 +37,16 @@ public BackgroundDocumentGeneratorTest()
 
         private HostProject HostProject2 { get; }
 
+        [Fact]
+        public void TryParse()
```

Comment:
Definitely need a better test name. No idea what this is for.

Reply:
Don't actually need that test, it was a relic of me trying to figure out a different issue.